src/main/java/org/mate/bookstorespringboot/controller/BookController.java

```diff
@@ -25,8 +26,8 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public ResponseEntity<List<BookDto>> getBooks() {
-        return ResponseEntity.ok(bookService.findAll());
+    public ResponseEntity<List<BookDto>> getBooks(Pageable pageable) {
+        return ResponseEntity.ok(bookService.findAll(pageable));
```

Comment:
```suggestion
    public List<BookDto> getBooks(Pageable pageable) {
        return bookService.findAll(pageable);
```
you can avoid wrapping to ResponceEntity. Also, it is better to return a page instead  list, because it contains information about total amount of objects and number of page